commands/serverless.js

```diff
@@ -10,6 +10,8 @@ module.exports = function (input, flags, logger, options) {
   const cwd = options.cwd
   const out = flags.out
   const env = flags.env
+  const vpcSecurityGroups = flags.vpcSecurityGroups
+  const vpcSubnets = flags.vpcSubnets
```

Comment:
Ah, yes, that does work. The quotes definitely work (just tested it).

Well, we don't have the spaces at all over in SCS-22, so as long as we either have spaces with quotes or no spaces at all, this'll work just fine.